.changeset/few-peas-hunt.md

```diff
@@ -0,0 +1,31 @@
+---
+'@astrojs/markdoc': minor
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Update the internal `shiki` syntax highlighter to `shikiji`, an ESM-focused alternative that simplifies bundling and maintenance. There are no new options and changes to how you author code blocks and syntax highlighting.
```

Comment:
```suggestion
Updates the internal `shiki` syntax highlighter to `shikiji`, an ESM-focused alternative that simplifies bundling and maintenance. 

There are no new options and no changes to how you author code blocks and syntax highlighting.
```

I'm now enforcing a standard verb tense here! Add your "s"s, everyone!  :smile: 

Also splitting this out thinking of the auto-generated blog post heading from the first line. So for minors, if there's a good one-line headline, let's keep those separated out.